Ten Elephants/Ten Elephants/Cache/CachedImageFetcher.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  CachedImageFetcher.swift
+//  Ten Elephants
+//
+//  Created by Kirill Denisov on 13.12.2021.
+//
+
+import UIKit
+import Foundation
+
+class CachedImageFetcher {
+    
+    private enum Constants {
+        static let cacheCountLimit: Int = 25
+    }
+    
+    init() {
+        self.cachedImages.countLimit = Constants.cacheCountLimit
+    }
+    
+    init(countLimit: Int) {
```

Comment:
Можно убрать конструктор без параметров и написать
```suggestion
    init(countLimit: Int = Constants.cacheCountLimit) {
```

Reply:
а почему так лучше?